web/server/vue-cli/src/views/NewFeatures.vue

```diff
@@ -1595,7 +1595,7 @@ analyzer:
         </new-release-item>
       </v-timeline-item>
 
-      <v-timeline-item fill-dot icon="mdi-star" color="green lighten-1">
+      <v-timeline-item fill-dot icon="mdi-star">
```

Comment:
One of the stars was incorrectly green, while the corresponging highlight was blue. It should look like this:
![image](https://user-images.githubusercontent.com/23276031/232026271-d0b73f03-2f0d-45cf-bddf-f5e21fd0e296.png)
![image](https://user-images.githubusercontent.com/23276031/232026305-c45dce52-56c7-4325-abeb-2d98e963419c.png)
Not mixed up:
![image](https://user-images.githubusercontent.com/23276031/232026491-88928dfb-8e80-4e54-ac17-48960cd10838.png)